.github/dependabot.yml

```diff
@@ -5,8 +5,20 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
-    # we have too many dependency PRs today
+    # we have too many dependency PRs today to rebase automatically
     rebase-strategy: "disabled"
+    groups:
+      babel:
+        patterns:
+          - "@babel/*"
+          - "babel-loader"
+      typescript-eslint:
+        patterns:
+          - "@typescript-eslint/*"
+      vite:
+        patterns:
+          - "vite"
+          - "@vitejs/*"
```

Comment:
I don't think this should be in your PR, looks like your merge went haywire

Reply:
What should I do now to resolve this😅?? But unit tests are running fine and there is no merge conflicts right but where the problem lies?

Reply:
it's gone after I merge from main